common/src/BasePlugin.php

```diff
@@ -218,4 +218,29 @@ private function isUniqueAttribute(array $array, string $value): bool {
     private function isValidGtin(string $value): bool {
         return preg_match('/^\d{8}(?:\d{4,6})?$/', $value);
     }
+
+    public function getManualSyncAction(): string {
```

Comment:
Since all of these functions are specific to the WooCommerce integration, don't they belong in that class?